app/web/server.rb

```diff
@@ -46,28 +46,24 @@ def json_params
 
     post '/todo/:todo_id' do
       command = Commands::Todo::Add::Command.new(json_params)
```

Comment:
Yeah I prefer that. We don't do anything based on the result of command validation in command handlers so it avoids the repetition and chance of forgetting to validate in command handlers.